src/builder/form/TestOptions.js

```diff
@@ -46,17 +67,10 @@ function TestOptions(props) {
           onChange={(e) => handleChange(e, 0)}
         /> : null
       }
-      <UploadFile
```

Comment:
What UploadFile component is this removing?

Reply:
This is the duplicate button that was in the maker object. I removed it and tested, I can re-add it and take care of it in another story if you think the two should be separate.

Reply:
I think it's good to remove here, just wanted to make sure it was the duplicate one and not one currently being used.